source/_patterns/04-pages/article--research~with-content-aside-with-scroll.json

```diff
@@ -311,7 +311,11 @@
             }
           }
         }
-      ]
+      ],
+      "seeMoreLink": {
+        "name": "Further reading",
+        "url": "#"
```

Comment:
Ditto to previously, I don't know where this URL comes from, maybe I'm missing some key understanding?